app/views/users/new.html.erb

```diff
@@ -1,6 +1,13 @@
 <div class="container">
   <h3>Create a New Account</h3>
   <%= form_for(@user) do |f| %>
+    <% if @user.errors.any? %>
```

Comment:
Should we pull this out to a partial to re-use for future edit-user functionality?